assets/stylesheets/parent_group_overview.scss

```diff
@@ -0,0 +1,12 @@
+.parent_group_overview_grouping_active {
```

Comment:
Most of our scss files as well as all the Perl code use 4 spaces. I'd prefer to use 4 spaces in new files as well (for the sake of consistency).

Reply:
noticed and wip